src/igtcloud/client/tools/common.py

```diff
@@ -47,11 +51,13 @@ def find_institute_by_name(project_id: str, institute_name: str) -> Optional[Ins
 def find_project_and_institutes(project_name: str, institute_name: str = None) -> Tuple[Optional[Project], List[Institute]]:
     institutes = list()
     project = find_project_by_name(project_name)
```

Comment:
Will find_project_by_name return None or raise an error in case of no permissions?

Reply:
It currently returns `None` values if the data can't be fetched, the caller of the function handles the errors.